agent/helpers.go

```diff
@@ -98,11 +97,12 @@ func (h Helper) isRouteExist(ip net.IP, netmask string) error {
 }
 
 // createRoute creates IP route, returns nil if success and error otherwise.
-func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string) error {
+func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string, extraArgs ...string) error {
```

Comment:
The information already leaks out to the caller with the "via" parameter that needs to match a keyword used by the ip command. I was trying to keep the impact as minimal as possible.

It's a function used in two places with two relatively different needs.
It might be better if it we just done directly in those two places, instead of making this function more multi-purposed with a handful of constants.